tools/extract/extract_genomic_dna.py

```diff
@@ -35,17 +35,31 @@ def reverse_complement( s ):
     return "".join( reversed_s )
 
 def check_seq_file( dbkey, GALAXY_DATA_INDEX_DIR ):
+    # Checks for the presence of *.nib files matching the dbkey within alignseq.loc
     seq_file = "%s/alignseq.loc" % GALAXY_DATA_INDEX_DIR
     seq_path = ''
     for line in open( seq_file ):
         line = line.rstrip( '\r\n' )
         if line and not line.startswith( "#" ) and line.startswith( 'seq' ):
             fields = line.split( '\t' )
-            if len( fields ) < 3:
-                continue
-            if fields[1] == dbkey:
+            if len( fields) >= 3 and fields[1] == dbkey:
+                print "Using *.nib genomic reference files"
                 seq_path = fields[2].strip()
                 break
```

Comment:
You can use a `return` instead of `break`, so then you don't need the next `if`.